packages/sdk-v2/src/contexts/goodid/GoodIdContext.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { createContext } from "react";
+
+import type { VerifiableCredential } from "../../sdk/goodid/types";
+import { createCredentialsDb, useCredentials } from "../../sdk/goodid";
+
+type IGoodIdContext = {
+  createCredential?: (credential: VerifiableCredential) => Promise<void>;
+  getActiveCredentials?: () => Promise<VerifiableCredential[] | undefined>;
+  db: any;
+};
+
+export const GoodIdContext = createContext<IGoodIdContext>({
+  db: undefined
+});
+
+interface IGoodIdContextProvider {
+  children: any;
+  localDb?: any; //todo: define db interface for wallet
+}
+
+export const GoodIdContextProvider = ({ children, localDb }: IGoodIdContextProvider) => {
```

Comment:
```suggestion
export const GoodIdContextProvider: FC<PropsWithChildren<IGoodIdContextProvider>> = ({ children, localDb }) => {
```

(import types from react)

Reply:
done